exercise-src/README.md

```diff
@@ -0,0 +1,66 @@
+# Academy Generator
```

Comment:
I’m move this to the main README so it’s discoverable.

Reply:
I added a "Generating Exercises" section to the main readme that links here, but there's a lot of info here that's not relevant to most courses, so I think its better to leave the majority of it here.